gptables/core/api.py

```diff
@@ -50,6 +52,12 @@ def produce_workbook(
     auto_width : bool, optional
         indicate if column widths should be automatically determined. True
         by default.
+    gridlines : int, optional
+        option to hide or show gridlines on worksheets. 0 - don't hide gridlines, 
+        1 - hide printed gridlines only, or 2 - hide screen and printed gridlines.
```

Comment:
I know these options are used in xlsxwriter but I would like to make it more explicit, eg
`show_all` instead of `0`
`hide_printed` instead of `1`
`hide_all` instead of `2`
It might be worth discussing whether we also keep the numbers for consistency with xlsxwriter